src/app/map/miyagi/all/page.tsx

```
@@ -0,0 +1,271 @@
+  });
+}
+
+async function loadVoteVenuePins(layer: any, L: any, area:string | null = null) {
+  const pins = await getVoteVenuePins(area);
+  const grayIcon = createGrayIcon(L);
+  pins.forEach(pin => {
+    const marker = L.marker([pin.lat, pin.long], {
+      icon: grayIcon
+    }).addTo(layer);
+
+    const escape = (s: string) => s.replace(/[&<>"'`]/g, c =>
```

Biome (1.9.4)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)